docs/0.2-DRAFT/index.md

```diff
@@ -131,28 +127,32 @@ If present `ro-crate-preview.html` MUST:
   ```
 
 `ro-crate-preview.html` SHOULD:
-  *   If there is sufficient metadata, contain a prominent _“Cite-as”_ text with a natural language datacite citation.
+  *   Contain at least the same information as the _RO-Crate JSON-LD_, with the exception that files which have no description, creator or similar metadata MAY not be listed in the website.
+  *   Display at least the metadata relating to the _Root Data Enity_ as static HTML without the need for scripting. It MAY contain extra features enabled by JavaScript.
+  *   When a _Data Entity_ or _Context Entity_ is referenced by its ID:
+    *   If it has a [name](http://schema.org/name) property, provide a link to its HTML version.
+    *   If it does not have a name (e.g. a [GeoCoordinates](http://schema.org/GeoCoordinates) location), show it embedded in the HTML for the entity.
+  *   For keys that resolve in the `RO-Crate JSON-LD Context` to a URI, indicate this (the simplest way is to link the key to its definition.
+  *   For external URI values, provide a link.
+  *   If there is sufficient metadata, contain a prominent _“Cite-as”_ text with a natural language data citation (see for example the [FORCE11 Data Citation Principles](https://www.force11.org/group/joint-declaration-data-citation-principles-final)).
   *   If there are additional resources necessary to render the preview (e.g. CSS, JSON, HTML), link to them in a subdirectory `ro-crate-preview-files/`
```

Comment:
@ptsefton I'm not sure if all these SHOULDs still apply? They are taken from the description of RO Website which was at the bottom of the spec.

Reply:
We should keep them in one place.. perhaps link to the other section.

Reply:
I think these are a bit detailed, and in #30 we agreed to simplify this list of requirements for `ro-crate-preview.html`

Reply:
They are all in one place now as part of this PR. I agree they are a bit detailed but I didn't want to lose them in case @ptsefton feels they are required.